tests/devapps/XForms/XForms/App.xaml.cs

```diff
@@ -42,13 +42,17 @@ public partial class App : Application
         public const string DefaultClientId = "4b0db8c2-9f26-4417-8bde-3f0e3656f8e0";
         // For system browser
         //public const string DefaultClientId = "5a434691-ccb2-4fd1-b97b-b64bcfbc03fc";
+
         public const string B2cClientId = "e3b9ad76-9763-4827-b088-80c7a7888f79";
 
+
         public static string RedirectUriOnAndroid = Microsoft.Identity.Client.Core.Constants.DefaultRedirectUri; // will not work with system browser
         // For system browser
         //public static string RedirectUriOnAndroid = "msauth-5a434691-ccb2-4fd1-b97b-b64bcfbc03fc://com.microsoft.identity.client.sample";
         
-        public static string RedirectUriOnIos = Microsoft.Identity.Client.Core.Constants.DefaultRedirectUri;
+        public const string BrokerRedirectUriOnIos = "msauth.com.yourcompany.XForms://auth";
```

Comment:
aside (another PR): should we have tools in the builders that helps building this? and perhaps also the regular redirecturi?

Reply:
will add to todo somewhere.

---
In reply to: [258178862](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/881#discussion_r258178862) [](ancestors = 258178862)